apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/Logging/Logger.cs

```diff
@@ -77,7 +77,7 @@ public static void SetLogger(Logger instance)
         /// <summary>
         /// 
         /// </summary>
-        public static int PerformanceTraceLogInterval { get; set; } = 30000;
+        public static TimeSpan PerformanceTraceLogInterval { get; set; } = TimeSpan.Zero;
```

Comment:
yes though I may at some point change it so that the logger stops and restarts the loop altogether if someone toggles perf logging.  But since this is internal, I don't see much point in making this completely optimal.